test/unit/formats.spec.ts

```diff
@@ -356,9 +376,7 @@ describe('formats', () => {
             .withParameter(pathParameterBuilder.withInt64Named('value'));
 
         const minimumInt64Allowed = '-9223372036854775808';
-        const minimumInt64AllowedMinusOne = '-9223372036854775809';
         const maximumInt64Allowed = '9223372036854775807';
-        const maximumInt64AllowedPlusOne = '9223372036854775808';
```

Comment:
See https://github.com/ajv-validator/ajv-formats/pull/22#issuecomment-808896823

TLDR: JSON only supports doubles, and JS (without resorting to BigInt) too; pretending otherwise is pointless.